package.json

```diff
@@ -82,6 +82,7 @@
     "@testing-library/react-hooks": "^7.0.2",
     "@testing-library/user-event": "^14.2.0",
     "@types/jest": "^27.4.1",
+    "@types/lodash": "^4.14.186",
```

Comment:
Why are we adding this as a dependency when there doesn't appear to be any `lodash` dependency, e.g. `lodash` is not listed as a dependency?